playroom/snippets.tsx

```diff
@@ -1048,9 +1069,10 @@ const navigationBarSnippets = [
         </Badge>
       </NavigationBarAction>
       <NavigationBarAction onPress={() => {}} aria-label="Open profile">
-        <Circle
-          backgroundImage="https://i.pravatar.cc/100?img=31"
+        <Avatar
           size={isDesktopOrBigger ? 32 : 24}
+          initials="ML"
+          src="https://i.pravatar.cc/100?img=31"
```

Comment:
does it make sense to have `intials` and `src` at the same time?

Reply:
Yes, if the image fails, it fallbacks to initials

https://github.com/Telefonica/mistica-web/blob/master/src/avatar.tsx#L104